xbmc/peripherals/devices/PeripheralCecAdapter.cpp

```diff
@@ -175,7 +175,7 @@ void CPeripheralCecAdapter::Announce(AnnouncementFlag flag, const char *sender,
   else if (flag == GUI && !strcmp(sender, "xbmc") && !strcmp(message, "OnScreensaverActivated") && m_bIsReady)
   {
     // Don't put devices to standby if application is currently playing
-    if ((!g_application.m_pPlayer->IsPlaying() && !g_application.m_pPlayer->IsPaused()) && m_configuration.bPowerOffScreensaver == 1)
+    if (!g_application.m_pPlayer->IsPausedPlayback() && m_configuration.bPowerOffScreensaver == 1)
```

Comment:
IsPlaybackPaused() is not new
This code here only worked by chance because IsPaused has ever been undefined in case of playback not being active.

> Don't we have an alternative in our beloved CApp that tells whether the player is active?

yes, we do:
g_application.m_pPlayer->IsPlaying()

IsPlaying does not give any information on whether playback is paused.